src/fontra/client/css/core.css

```diff
@@ -18,6 +18,7 @@
 :root {
   /* https://css-tricks.com/the-css-custom-property-toggle-trick/ */
   --fontra-theme-marker: ; /* a space, which is valid */
+  --menu-top-bar-height: 35px;
```

Comment:
We have a "top bar" that contains a "menu bar". "menu top bar" makes no sense.

Reply:
ok, before my changes, they were called `--editor-top-bar-XXX`. Should I rename it to `--top-bar-XXX`? 
For me it's not clearly separated from each other, for example `--editor-top-bar-link-hover` was used in the css classes `.menu-item.current {`, which would be more about "menu bar", then "top bar".
Let me think about it. Maybe there is more about the whole idea of separating between "top bar" and "menu bar".

Reply:
Yes, there was indeed already some confustion between "top bar" and "menu bar". Moving some of these to the menu-bar module fixes most of this, no?

I would name this particular variable `--top-bar-height`.

Reply:
Done:
- Removed "top" from "--menu-top-bar-XXX" as it belongs to menu-bar only
- Removed unused "menu-top-bar-border-color"
- "--top-bar-height"